src/Fetch.js

```diff
@@ -0,0 +1,186 @@
+import { useEffect, useContext } from 'react'
+import { useEnv } from './ui/EnvContext'
+import ToasterContext from './ui/ToasterContext'
+import {extractLinks, extractPagination} from "./ui/utils"
+
+function requestHeaders() {
+  const token = window.localStorage.getItem('accessToken')
+  const client = window.localStorage.getItem('client')
+  const uid = window.localStorage.getItem('uid')
+
+  return {
+    'Api-Version': 'v1',
+    'Access-Token': token,
+    'Authorization': `Bearer ${token}`,
+    'Client': client,
+    'Content-Type': 'application/json',
+    'Token-Type': 'Bearer',
+    'Uid': uid
+  }
+}
+
+async function fetchData(resource) {
```

Comment:
I moved everything from the react app. But we are using only the function `useFetch`. I can see how these might come in handy, but I can also see how this is a case of You Aren't Gonna Need It. Which way do we want to go?